scripts/userstyles.yml

```
@@ -638,14 +641,14 @@ userstyles:
     readme:
       app-link: "https://search.nixos.org"
     current-maintainers: [*cecelot]
-  nixos.wiki:
+  wiki.nixos.org:
     name: NixOS Wiki
     categories: [wiki, productivity]
```

```suggestion
    categories: [wiki, development]
```

Seems more apt even for the previous wiki :)